### DISCUSSION TITLE
Material Dark Full (portrait) [3.5"]

### DISCUSSION THREAD

**Discussion by geekm0nkey** [mathoudebine/turing-smart-screen-python#284] [Themes]
This is a remix of the theme found [here](https://github.com/mathoudebine/turing-smart-screen-python/discussions/276). 

I wanted to make use of the WHOLE tiny screen, so removed 99% of the empty space, added some flare around the gauge boxes, more colors, and made everything bigger. (I'm old) All labels were made generic, really no need for me to spell out every product I use. I have also provided the Photoshop document so you can edit the background, colors, frames, borders, etc to your liking.

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/35175147/63852c8c-cbdf-4094-98cb-fd27460074ab)

**Download**
[Material_Dark_Full_Portrait.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12016915/Material_Dark_Full_Portrait.zip)

**Comment by DJRoby19** [comment]
Well done beautiful work
could you have the landscape version?

**Comment by geekm0nkey** [comment]
I quickly threw together a landscape version per request. PSD is provided should you want to change things,

![screencap](https://github.com/mathoudebine/turing-smart-screen-python/assets/35175147/4c09d87b-0151-4334-9d63-16d87bdd3223)

**Download**
[Material_Dark_Full_Landscape.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12089443/Material_Dark_Full_Landscape.zip)

**Reply by DJRoby19** [reply]
Wow thank you so much
If I change something, I'll post it

**Comment by Freekers** [comment]
My favorite theme (so far) ! Thank you

**Comment by a-sber-rh** [comment]
Hi, I make a "remix" of your desing. I want to ask if I can upload it. 
Thanks.

**Comment by geekm0nkey** [comment]
Sure.. Be my guest. 


⁣Get BlueMail for Android ​

**Comment by SuperWaper** [comment]
Hello, no stat for GPU, any idea ?
It works with "basics" theme in initial bundle

**Reply by c0ldJS** [reply]
Is your operating system and card combo listed with a ✅ or ❕on [the wiki](https://github.com/mathoudebine/turing-smart-screen-python/wiki/System-monitor-%3A-hardware-support)?

**Reply by SuperWaper** [reply]
Yes i think so, i have an Intel i9 and NVIDIA 3080.
It's works with all other themes

**Reply by mathoudebine** [reply]
There is an issue with the theme file: the INTERVAL for the GPU sensors should be common to all:
<img width="226" height="97" alt="image" src="https://github.com/user-attachments/assets/530de183-ced8-4317-85bc-fe2bcd7a7fe1" />
should instead be
<img width="237" height="80" alt="image" src="https://github.com/user-attachments/assets/8f51df36-03e5-45ff-88be-bc1a51a4318b" />

**Reply by mathoudebine** [reply]
I just added this info to the wiki: https://github.com/mathoudebine/turing-smart-screen-python/wiki/Troubleshooting#all-os-my-cpugpuother-hardware-is-not-displayed--some-data-are-missing

**Comment by Adzzzzzzzzz** [comment]
Hi, I used your template and changed a few things. Just sharing my version. 

[
<img width="320" height="480" alt="preview" src="https://github.com/user-attachments/assets/240f3bf8-2729-456d-a7d3-41d83e8b53c7" />
Material_Dark_Full_Portrait.zip](https://github.com/user-attachments/files/25474688/Material_Dark_Full_Portrait.zip)
